src/imitation/scripts/config/data_collect.py

```diff
@@ -104,3 +104,10 @@ def walker():
 def fast():
   """Intended for testing purposes: small # of updates, ends quickly."""
   total_timesteps = int(1e4)
+
+
+@data_collect_ex.named_config
+def mnist():
+  env_name = "imitation/Mnist-v0"
+  total_timesteps = int(1e4)
```

Comment:
This seems far too small to get a good policy -- intentional?

Reply:
Used it when debugging, nice catch. Updating it to `1e6` to get a 99% optimal policy.